### DISCUSSION TITLE
curiosity

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#125] [General]
Just for my curiosity is possible after install Spirallinux transform it in "pure debian"?

**Comment by geckolinux** [comment]
Hi there, thanks for the question. SpiralLinux already is pure Debian. An important difference between SpiralLinux and other Debian-based distros is that SpiralLinux directly uses the Debian repositories and unmodified Debian packages.

**Reply by robertocannito** [reply]
Thank you Sam, this was just for understand better difference with them (debian) I love spirallinux in every case, I hope that you will continue to support this great distro.

**Reply by robertocannito** [reply]
another question, do you thing that will be possible install also snapper-btrfs on this?
this is a great good tool for btrfs

**Reply by geckolinux** [reply]
> snapper-btrfs

Hmm, do you have a link perchance?

**Reply by robertocannito** [reply]
here   https://gitlab.com/btrfs-assistant/btrfs-assistant

installed following instruction for Debian but it doesn't take any snap

**Reply by geckolinux** [reply]
It's hard to say why it's not working, but since it was created for Garuda it might be expecting a different subvolume layout.
By the way, the Snapper GUI tool included in SpiralLinux does most of what btrfs-assistant does.
